contracts/DecentralizedKV.sol

```diff
@@ -4,16 +4,21 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import "./MerkleLib.sol";
 import "./BinaryRelated.sol";
-import "./EthStorageConstants.sol";
```

Comment:
We should also have a test gas cost result for putBlob func: case 1: putBlob for a new key; case 2: update an existing key

Reply:
need a test gas cost for updating an existing key value